content-security-policy/connect-src/connect-src-json-import-allowed.sub.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
I think the logTest and alertAssert logic were used to autoconvert test when WPTs were first created, but we prefer writing new tests in a more direct way.

What about simplifying as

(using the fact that a [promise_test](https://web-platform-tests.org/writing-tests/testharness-api.html?highlight=promise_test#promise-tests) succeeds or fails depending on whether the returned promise does)

```
<!DOCTYPE html>
<html>

<head>
    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' http://{{host}}:{{ports[http][0]}}; script-src 'self' 'unsafe-inline';">
    <script src="/resources/testharness.js"></script>
    <script src="/resources/testharnessreport.js"></script>
</head>

<body>
    <script>

        promise_test(async t => {
          window.addEventListener('securitypolicyviolation', t.step_func(e => {
              assert_unreached("No security policy violation should be raised.");
          }));
          return import("./resources/dummy.json", { with: { type: "json" } });
        }, "import should be allowed");
    </script>
</body>

</html>
```